### PR TITLE
Added Member.owner

### PR DESCRIPTION
**Pull request by OrangutanGaming**

### CODE REVIEW

**Comment by MusicOnline** [comment]
Eh, you can compare using `Guild.owner` already.
Also, `Member.owner` is misleading. At least make it `Member.is_owner`.

Still, I don't think this is needed.

**Comment by laggycomputer** [comment]
Agreed. I cannot think of many uses for this.

File: discord/member.py [hunk 1]
```diff
@@ -381,7 +386,7 @@ def guild_permissions(self):
         administrator implication.
         """
 
-        if self.guild.owner == self:
+        if self.owner:
```
Comment (Helehelehele) [inline discord/member.py]: You forgot to update this.

Also, ideally, you'd rebase your commits.

**Comment by laggycomputer** [comment]
This makes an already very short comparison even shorter, and even potentially confusing.

**Pull request closed without merging**